RetroBar/Languages/српски.xaml

```diff
@@ -0,0 +1,87 @@
+    <s:String x:Key="multiple_displays">Више екрана</s:String>
+    <s:String x:Key="show_tasks_on">Прика_жи прозоре на:</s:String>
+    <x:Array x:Key="show_tasks_on_values" Type="s:String">
+        <s:String>Свим тракама задатака</s:String>
+        <s:String>Траци задатака на којој је прозор</s:String>
+        <s:String>Траци задатака на којој је прозор и на главној</s:String>
+    </x:Array>
+    <s:String x:Key="version">Верзија {0}</s:String>
+    <s:String x:Key="visit_on_github">Отворите GitHub</s:String>
+    <s:String x:Key="ok_dialog">ОК</s:String>
+
+    <s:String x:Key="customize_notifications">Прилагоди обавештења</s:String>
+    <s:String x:Key="customize_notifications_info">RetroBar приказује иконе за активна и хитна обавештења, а неактивна сакрива. Можете да промените ово понашање за ставке на списку испод.</s:String>
+    <s:String x:Key="customize_notifications_instruction">Изаберите ставку, па изаберите понашање њених обавештења:</s:String>
+    <s:String x:Key="hide_when_inactive">Сакриј када је неактивно</s:String>
+    <s:String x:Key="always_show">Увек прикажи</s:String>
+    <s:String x:Key="name_heading">Име</s:String>
+    <s:String x:Key="behavior_heading">Понашање</s:String>
+
+    <s:String x:Key="start_text">Старт</s:String>
+    <s:String x:Key="start_text_xp">старт</s:String>
+    <s:String x:Key="start_button_tip_98">Кликните овде да почнете.</s:String>
+    <s:String x:Key="start_button_tip">Кликните овде да почнете</s:String>
+    <s:String x:Key="start_button_tip_vista">Старт</s:String>
+
+    <s:String x:Key="retrobar_title">RetroBar трака задатака</s:String>
+    <s:String x:Key="toolbars">_Траке са алаткама</s:String>
+    <s:String x:Key="quick_launch">_Брзо покретање</s:String>
+    <s:String x:Key="new_toolbar">_Нова трака са алаткама...</s:String>
+    <s:String x:Key="show_taskman_2k">Управљач _задацима...</s:String>
+    <s:String x:Key="show_taskman">Управљач _задацима</s:String>
+    <s:String x:Key="tray_properties">С_војства</s:String>
+    <s:String x:Key="update_available">_Ажурирање доступно...</s:String>
+    <s:String x:Key="exit_retrobar">_Напусти RetroBar</s:String>
+    <s:String x:Key="customize_notifications_menu">Прилагоди _обавештења...</s:String>
+
+    <s:String x:Key="restore">_Врати у претходно стање</s:String>
+    <s:String x:Key="move">Преме_сти</s:String>
+    <s:String x:Key="size">Вели_чина</s:String>
+    <s:String x:Key="minimize">Ум_ањи</s:String>
+    <s:String x:Key="maximize">_Увећај</s:String>
+    <s:String x:Key="close">_Затвори</s:String>
+
+    <s:String x:Key="show_hidden">Прикажи скривене иконе</s:String>
+    <s:String x:Key="hide">Сакриј</s:String>
+
+    <s:String x:Key="set_time">_Подесите датум и време</s:String>
+
+    <s:String x:Key="open_folder">Отвори &amp;фасциклу</s:String>
+
+    <FlowDirection x:Key="flow_direction">LeftToRight</FlowDirection>
+
+    <FontFamily x:Key="start_text_xp_font_family">Франклин Gothic</FontFamily>
```

Comment:
The font seems to have been accidentally translated, it should be "Franklin Gothic"

Reply:
Oops! I automatically translated between scripts and didn't notice that. Should be fixed now.